content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -71,42 +113,80 @@ set(FILES
 )
 ```
 
+### Adding a temporary auto-component
+{{< note >}}
+There's currently a [bug](https://github.com/o3de/o3de/issues/4058) causing a build failure if multiplayer auto-components are enabled, but no auto-components are created. As a work-around, create a temporary auto-component.
+{{< /note >}}
+1. Create a new folder under your project's `Code\Source\` directory called `AutoGen`. 
+    {{< note >}}This AutoGen directory doesn't have to be temporary. All future multiplayer auto-components can live here.{{< /note >}}
+1. Create a new, temporary auto-component file under `Code\Source\AutoGen` called `MyFirstNetworkComponent.AutoComponent.xml`.
+    {{< note >}}The name of this multiplayer auto-component can be anything, it's just important to ensure the name is used consistently. For this guide we will use "MyFirstNetworkComponent".{{< /note >}}
+
+1. Modify `Code\Source\AutoGen\MyFirstNetworkComponent.AutoComponent.xml` to have the following content:
+    ```xml  
+    <?xml version="1.0"?>
+
+    <Component
+        Name="MyFirstNetworkComponent"
+        Namespace="<ProjectName>"
+        OverrideComponent="false"
+        OverrideController="false"
+        OverrideInclude=""
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    </Component>
+    ```
+    {{< important >}}Make sure the "Namespace" property is wrapped in quotes and matches the name of your project.
+    {{< /important >}}
+1. Register the temporary auto-component with CMake by updating `<projectname_files.cmake>`
+    ```cmake
+    set(FILES
+        ...
+        Source/AutoGen/MyFirstNetworkComponent.AutoComponent.xml
+    )
+    ```
+{{< note >}}Feel free to delete this temporary auto-component, or use it as a starting point, once you're ready to build further multiplayer components. Find out more in general about multiplayer auto-components [here](/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autocomponents/), or follow [this tutorial](/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component/).{{< /note >}}
```

Comment:
Maybe "After completing the setup steps in this guide, you can delete the temporary auto-component, or use it.."?

Reply:
Updated! Also reiterated the fact that at least 1 auto-component must exist in case anyone mistake that sentence as saying you can delete the temp-auto-component without first adding an new one.